content/articles/creating-an-animated-javafx-field-validation-alert/index.md

```diff
@@ -0,0 +1,758 @@
+
+The AnchorPane will have the following:
+
+- **Style**
+- - `-fx-text-fill`:  #8ecae6
+- **Layout**
+- - `Pref Width`: 400
+- - `Pref Height`: 600
+
+The layout makes it fit perfectly with the scene initially created.
+
+- Drag and drop a label from the controls tab into the AnchorPane in the center of the SceneBuilder.
+
+The label will have the following:
+
+- **Properties**
+- - `Text`: LOGIN
+- - `Font`: System 25px Bold
+- **Style**
+- - `-fx-text-fill`:  #219ebc
+- **Layout**
+- - `Pref Width`: 100
+- - `Pref Height`: 20
+- - `Layout X`: 150
+- - `Layout Y`: 40
+
+When viewed using the '**Text**' view, the following code will be found to be automatically generated:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.image.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.shape.*?>
+<?import javafx.scene.text.*?>
+
+<AnchorPane maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="600.0" prefWidth="400.0" style="-fx-background-color: #8ecae6;" xmlns="http://javafx.com/javafx/11.0.2" xmlns:fx="http://javafx.com/fxml/1" fx:controller="authentication.login.HelloController">
+  <!-- Contents inside the XML file -->
+   <children>
+     <!-- Title -->
+      <Label layoutX="150.0" layoutY="40.0" prefHeight="20.0" prefWidth="100.0" style="-fx-text-fill: #219ebc;" text="LOGIN">
+         <font>
+            <Font name="System Bold" size="25.0" />
+         </font></Label>
+   </children>
+</AnchorPane>
+```
+
+- Head over to this [link](https://github.com/justusmbuvi/Creating-an-Animated-JavaFX-field-Validation-alert/blob/main/src/main/resources/authentication/login/panda.png) and download the panda image. This will be placed to beautify the scene.
+- Once downloaded, copy and paste it into the location of the FXML file.
+- Drag and drop an ImageView control into the AnchorPane. The control will hold the image.
+- Set the following to the ImageView control:
+
+
+- **Properties**
+- - `Image`: panda.png
+- - `Font`: System 25px Bold
+- **Layout**
+- - `Fit Width`: 300
+- - `Fit Height`: 300
+- - `Layout X`: 90
+- - `Layout Y`: 57
+
+The Scene Builder generates the code shown below:
+
+```xml
+      <ImageView fitHeight="300.0" fitWidth="300.0" layoutX="90.0" layoutY="57.0" pickOnBounds="true" preserveRatio="true">
+         <image>
+            <Image url="@panda.png" />
+         </image>
+         <viewport>
+            <Rectangle2D />
+         </viewport>
+      </ImageView>
+```
+
+- Now, add a Rectangle from the Shapes category. It shall have the following:
+
+
+- **Properties**
+- - `Arc Width`: 5
+- - `Fill`: #b9eaf8
+- - `Stroke`: #a3a3a3
+- - `Stroke Width`: 2
+- - `Stroke Line Cap`: Round
+- - `Fill`: BEVEL
+- **Layout**
+- - `Width`: 370
+- - `Height`: 320
+- - `Layout X`: 15
+- - `Layout Y`: 260

```

Comment:
If you already specified the values in the snippet there is no need of writing them separately. Do this for every instance you have a code snippet.